chapters/modules/chapter_8/pages/solidity_verifier.adoc

```diff
@@ -0,0 +1,37 @@
+[id="verifiers_solidity"]
+
+= Verifiers in Solidity
+
+L1 smart contract that verifies STARK proofs from SHARP.
+
+== Example of a Verifier transaction
+Let's take a look at a transaction that was sent to the Verifier contract on Ethereum mainnet. 
+https://etherscan.io/tx/0xde0bcb45698e8ef67fef8bff8615715dac0cf0882a131a21273e58607330d5f9[0xde0bcb45698e8ef67fef8bff8615715dac0cf0882a131a21273e58607330d5f9]
+
+=== Transaction trace
+- Verifier contract is a proxy contract so when it's called, implementation contract is called instead. Implementation contract is GpsStatementVerifier contract https://etherscan.io/address/0x6cb3ee90c50a38a0e4662bb7e7e6e40b91361bf6[0x6cB3EE90C50a38A0e4662bB7e7E6e40B91361BF6]
+- Next the GpsStatementVerifier contract executes opcode `CALL` to CpuFrilessVerifier contract https://etherscan.io/address/0xe9664d230490d5a515ef7ef30033d8075a8d0e24[0xe9664d230490d5a515ef7ef30033d8075a8d0e24]
```

Comment:
Building on the prior query, could we pinpoint the exact location in the code where the CALL operation to CpuFrilessVerifier is executed?

Reply:
added